diff-ex-and-viz.md

```diff
@@ -300,15 +300,13 @@ plotCounts(dds, gene=which.min(res$padj), intgroup="condition")
 What gene is plotted here (i.e., what criteria did we use to select a single 
 gene to plot)?
 
-### Transformation of sequencing-depth-normalized read counts
-
-While most models used for differential gene expression testing operate on the raw count values, many downstream analyses (including clustering & PCA) work better if the read counts are transformed to the log scale. While you will occasionally see log10 transformed read counts,log2 is more commonly used because it is easier to think about doubled values rather than powers of 10. **The transformation should be done in addition to sequencing depth normalization.**
+### Normalization & Transformation
 
 `DESeq2` automatically normalizes our count data when it runs differential 
 expression. However, for certain plots, we need to normalize our raw count data.
 One way to do that is to use the `vst()` function. It perform variance 
 stabilized transformation on the count data, while controlling for library
-size of samples. 
+size of samples. **The transformation should be done in addition to sequencing depth normalization.**
```

Comment:
VST does normalize for sequencing depth too :) that's what "controlling for library size means above"